app/adapters/alma_adapter.rb

```diff
@@ -48,9 +48,44 @@ def get_items_for_bib(id)
     bib_item_set.holding_summary
   end
 
+  # @param id [String]. e.g id = "991227850000541"
+  # @return [String] date record was created e.g. "2019-10-18Z"
+  def get_catalog_date(id)
```

Comment:
I think we want this logic in the `AlmaAdapter::MarcRecord` so that it adds the catalog date to a field that will be indexed by the logic added in https://github.com/pulibrary/marc_liberation/pull/945/files.

It should be a method called `enrich_with_catalog_date` or something like that, that we can then call from the bibliographic#update action once we've fetched the fields.

It's probably fine to just find it and put it in the `876d`; I don't think we need to worry about only using that for a value found in the bib (putting it in the `951w` if it came from the physical item and the `950b` if it came from the electronic item).

Reply:
Working on some ideas for this.

Reply:
Done. Take a look and let me know what you think.